cryodrgn/commands/abinit_homo.py

```diff
@@ -21,14 +20,6 @@
 
 logger = logging.getLogger(__name__)
 
-# def debug_signal_handler(signal, frame):
```

Comment:
Thanks for removing -- this was left in from debugging during development. In the future, we could think about adding a pdb tracer when the job is run in --verbose mode or something like that.